### PR TITLE
Commit Karate Base code

### PR DESCRIPTION
**Pull request by karthikn93**

### CODE REVIEW

File: src/test/java/configs/karate-config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function fn() {
```
Comment (roniltech) [inline src/test/java/configs/karate-config.js:1]: Add a folder for payloads with an example payload

File: src/test/java/performance/users.feature [hunk 2]
```diff
@@ -0,0 +1,46 @@
+Feature: sample karate test script
```
Comment (roniltech) [inline src/test/java/performance/users.feature]: Please remove this file and use the actual users file in performance

File: src/test/java/examples/users/UsersRunner.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package examples.users;
+
+import com.intuit.karate.junit5.Karate;
+
+class UsersRunner {
+    
+    @Karate.Test
+    Karate testUsers() {
```
Comment (roniltech) [inline src/test/java/examples/users/UsersRunner.java]: Please add other type of runner also

File: src/test/java/examples/ExamplesTest.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package examples;
```
Comment (roniltech) [inline src/test/java/examples/ExamplesTest.java]: This looks like a runner file
Please rename as ExamplesTestSuiteRunner.java

Comment (roniltech) [inline src/test/java/examples/ExamplesTest.java]: Please add an example feature file in this folder

File: src/test/java/configs/karate-config.js (hunk 1 above)
Comment (karthikn93) [reply src/test/java/configs/karate-config.js:1]: Added a folder and sample json

File: src/test/java/performance/users.feature (hunk 2 above)
Comment (karthikn93) [reply src/test/java/performance/users.feature]: Removed and referenced the actual file

File: src/test/java/examples/users/UsersRunner.java (hunk 3 above)
Comment (karthikn93) [reply src/test/java/examples/users/UsersRunner.java]: Added - ExamplesTestSuiteRunner.java

File: src/test/java/examples/ExamplesTest.java (hunk 4 above)
Comment (karthikn93) [reply src/test/java/examples/ExamplesTest.java]: Updated

Comment (karthikn93) [reply src/test/java/examples/ExamplesTest.java]: That's already available 
file name is 'users.feature'

It was actually under a sub-folder but now moved them in the same folder

**Review by roniltech — APPROVED** [review]

**Pull request merged**